CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+v.3.0.0-alpha
+=============
+
+- job.jobId changed to job.id.
+- refactored error messages into separate error module.
+- completed and failed job states are now represented in ZSETs.
```

Comment:
- job.jobId -> job.id  to reduce redundancy.
- error messages refactored so that we can unit test without needing to copy/paste the messages.
- ZSETs allows us to efficiently get ranges of completed and failed jobs, very useful for building UIs and scripts.